recipes-kernel/linux/linux-tb_5.13.bb

```diff
@@ -9,13 +9,10 @@ SRC_URI = "git://github.com/TrenchBoot/linux.git;protocol=https;branch=${KBRANCH
            git://git.yoctoproject.org/yocto-kernel-cache;type=kmeta;name=meta;branch=yocto-5.14;destsuffix=${KMETA} \
            file://defconfig \
            file://debug.cfg \
-           file://xen.cfg \
+           file://efi.cfg \
+           file://tpm2.cfg \
```

Comment:
This recipe needs to be refreshed to more recent kernel version (as new as possible from TB repo).

Reply:
@macpijan done in [54989a6](https://github.com/3mdeb/meta-trenchboot/pull/28/commits/54989a6273aa405df4e3f2cac13cab0c162ac91b)